terraform/amazon/templates/jenkins_user_data.yaml.template

```diff
@@ -104,12 +104,48 @@ write_files:
     [Service]
     MountFlags=shared
 
+- path: /etc/systemd/system/wing-tag.service
+  permissions: '0644'
+  content: |
+    [Unit]
+    Description=Tag Jenkins instance with public keys
+    After=network.target
+
+    [Service]
+    PermissionsStartOnly=true
+    Restart=on-failure
+    RestartSec=10
+    Environment=WING_CLOUD_PROVIDER=amazon
+{{- if .WingDevMode }}
+    Environment=WING_VERSION="${wing_version}"
+    ExecStartPre=/bin/sh -c 'aws s3 cp "s3://${wing_binary_path}" /opt/wing-$${WING_VERSION}/wing; chmod 0755 /opt/wing-$${WING_VERSION}/wing'
+{{- else }}
+    Environment=AIRWORTHY_VERSION=0.2.0
+    Environment=AIRWORTHY_HASH=2d69cfe0b92f86481805c28d0b8ae47a8ffa6bb2373217e7c5215d61fc9efa1d
+    Environment=WING_VERSION=0.5.3
+    ExecStartPre=/bin/sh -c '\
+      set -e ;\
+      test -x /opt/wing-$${WING_VERSION}/wing && exit 0 ;\
+      if [ ! -x /opt/airworthy-$${AIRWORTHY_VERSION}/airworthy ]; then \
+        mkdir -p /opt/airworthy-$${AIRWORTHY_VERSION} ;\
+        curl -sLo /opt/airworthy-$${AIRWORTHY_VERSION}/airworthy https://github.com/jetstack/airworthy/releases/download/$${AIRWORTHY_VERSION}/airworthy_$${AIRWORTHY_VERSION}_linux_amd64 ;\
+        echo "$${AIRWORTHY_HASH}  /opt/airworthy-$${AIRWORTHY_VERSION}/airworthy" | sha256sum -c ;\
+        chmod 755 /opt/airworthy-$${AIRWORTHY_VERSION}/airworthy ;\
+      fi ;\
+      /opt/airworthy-$${AIRWORTHY_VERSION}/airworthy download --output /opt/wing-$${WING_VERSION}/wing --sha256sums https://github.com/jetstack/tarmak/releases/download/$${WING_VERSION}/tarmak_$${WING_VERSION}_checksums.txt  --signature-armored https://github.com/jetstack/tarmak/releases/download/$${WING_VERSION}/tarmak_$${WING_VERSION}_checksums.txt.asc https://github.com/jetstack/tarmak/releases/download/$${WING_VERSION}/wing_$${WING_VERSION}_linux_amd64'
+ {{- end }}
+    ExecStart=/bin/sh -c 'exec /opt/wing-$${WING_VERSION}/wing tag --environment "${tarmak_environment}"'
+    Type=notify
```

Comment:
In theory it should just exit after it has finished, Type=notify seems wrong, should be more like Type=oneshot